howto/consume-topic/README.md

```diff
@@ -76,6 +76,27 @@ app.post('/orders', (req, res) => {
 app.listen(port, () => console.log(`consumer app listening on port ${port}!`))
 ```
 
+#### Subscribing on Kubernetes
+
+In Kubernetes, save the CRD to a file and apply it to the cluster:
```

Comment:
Custom Resources Definition (CRD), since first time CRD is used in the doc

Reply:
Changed (above this section, where CRD appears for the first time).